react/features/base/react/components/web/InlineDialogFailure.js

```diff
@@ -0,0 +1,75 @@
+import AKButton from '@atlaskit/button';
+import React, { Component } from 'react';
+
+import { translate } from '../../../i18n';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Inline dialog failure.
+ *
+ * @param {Object} props - The properties for the inline dialog.
+ * @returns {XML}
+ */
+class InlineDialogFailure extends Component {
+    /**
+     * {@code InlineDialogFailure}'s property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Allows to retry the call that previously didn't succeed.
+         */
+        onRetry: React.PropTypes.func,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: React.PropTypes.func
+    };
+
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {ReactElement}
+     */
+    render() {
+        const { t } = this.props;
+
+        const supportLink = interfaceConfig.SUPPORT_URL;
+        const supportLinkElem
+            = supportLink
+                ? ( // eslint-disable-line no-extra-parens
+                    <div className = 'inline-dialog-error-text'>
+                        <span>`{ t('inlineDialogFailure.supportMsg') } `</span>
```

Comment:
I think the backticks here just get displayed in the dialog, which might not be what you want.